rmap-loader-deposit-disco/pom.xml

```diff
@@ -183,14 +182,6 @@
     </dependency>
 
     <dependency>
-      <groupId>org.postgresql</groupId>
```

Comment:
I don't specifically remember! I was thinking about separating out the database stuff at one point and I also had some problems when using MySQL.  It's possible I did something in the middle of that and didn't undo it because nothing was complaining.  Is it needed? If so I can put it back.